package/contents/ui/lib/Card.qml

```diff
@@ -19,14 +19,18 @@ Rectangle {
         imagePath: root.enableTransparency ? "translucent/dialogs/background" : "opaque/dialogs/background"
         clip: true
         anchors.fill: parent
-        anchors.topMargin: rect.margins.top * root.scale
-        anchors.leftMargin: rect.margins.left * root.scale
-        anchors.rightMargin: rect.margins.right * root.scale
-        anchors.bottomMargin: rect.margins.bottom * root.scale
+        anchors.topMargin: -rect.inset.top + PlasmaCore.Units.smallSpacing * 1.5
+        anchors.leftMargin: -rect.inset.left  + PlasmaCore.Units.smallSpacing * 1.5
+        anchors.rightMargin: -rect.inset.right  + PlasmaCore.Units.smallSpacing * 1.5
+        anchors.bottomMargin: -rect.inset.bottom  + PlasmaCore.Units.smallSpacing * 1.5
```

Comment:
But now they'll not respect the scale of the applet? There's an option in the settings to increase or decrease the scale of the applet to make it larger or smaller.